gravitee-apim-portal-webui/src/app/components/gv-alert/gv-alert.component.ts

```diff
@@ -161,10 +182,32 @@ export class GvAlertComponent implements OnInit {
               value: AlertTimeUnit.HOURS,
             },
           ];
+          this.apiAllLabel = Object.values(translations)[5] as string;
+        });
+
+      this.subscriptionService
+        .getSubscriptions({
+          applicationId: this.application.id,
+          statuses: [StatusEnum.ACCEPTED, StatusEnum.PAUSED, StatusEnum.PENDING],
+          size: -1,
+        })
+        .toPromise()
```

Comment:
On L132 we are using Observable and here Promise, is there a rule to follow to be consistent across the codebase?